packages/models-library/src/models_library/api_schemas_webserver/folders_v2.py

```diff
@@ -16,17 +16,33 @@ class FolderGet(OutputSchema):
     folder_id: FolderID
     parent_folder_id: FolderID | None = None
     name: str
+
     created_at: datetime
     modified_at: datetime
     trashed_at: datetime | None
+    trashed_by: Annotated[
+        GroupID | None, Field(description="The primary gid of the user who trashed")
+    ]
     owner: GroupID
     workspace_id: WorkspaceID | None
     my_access_rights: AccessRights
 
-
-class FolderGetPage(NamedTuple):
-    items: list[FolderGet]
-    total: PositiveInt
+    @classmethod
+    def from_domain_model(
+        cls, folder_db: FolderDB, user_folder_access_rights: AccessRights
+    ) -> Self:
+        return cls.model_construct(
```

Comment:
please be very careful with using `model_construct` if anything you pass here is not of the type expected, it will not be converted.

Reply:
True